src/components/comment/comment_event.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import { CommonProps, keysOf } from '../common';
+import classNames from 'classnames';
+import { EuiI18n } from '../i18n';
+
+export type EuiCommentEventProps = CommonProps &
+  CommonProps & {
```

Comment:
```suggestion
  HTMLAttributes<HTMLDivElement> & {
```

and import `HTMLAttributes` from 'react'